GOVERNANCE.md

```diff
@@ -0,0 +1,114 @@
+
+## Overview
+
+**Velero**, an open source project, is committed to building an open, inclusive, productive and self-governing open source community focused on building a high quality tool that enables users to safely backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volumes. The community is governed by this document with the goal of defining how community should work together to achieve this goal.
+
+## Code Repositories
+
+The following code repositories are governed by Velero community and maintained under the `vmware-tanzu\Velero` namespace.
+
+* **[Velero](https://github.com/vmware-tanzu/velero):** Main Velero codebase
+* **[Helm Chart](https://github.com/vmware-tanzu/helm-charts/tree/master/charts/velero):** The Helm chart for the Velero server component
+* **[Velero CSI Plugin](https://github.com/vmware-tanzu/velero-plugin-for-csi):** This repository contains Velero plugins for snapshotting CSI backed PVCs using the CSI beta snapshot APIs
+* **[Velero Plugin for vSphere](https://github.com/vmware-tanzu/velero-plugin-for-vsphere):** This repository contains the Velero Plugin for vSphere. This plugin is a volume snapshotter plugin that provides crash-consistent snapshots of vSphere block volumes and backup of volume data into S3 compatible storage.
+* **[Velero Plugin for AWS](https://github.com/vmware-tanzu/velero-plugin-for-aws):** This repository contains the plugins to support running Velero on AWS, including the object store plugin and the volume snapshotter plugin
+* **[Velero Plugin for GCP](https://github.com/vmware-tanzu/velero-plugin-for-gcp):** This repository contains the plugins to support running Velero on GCP, including the object store plugin and the volume snapshotter plugin
+* **[Velero Plugin for Azure](https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure):** This repository contains the plugins to support running Velero on Azure, including the object store plugin and the volume snapshotter plugin
+* **[Velero Plugin Example](https://github.com/vmware-tanzu/velero-plugin-example):** This repository contains example plugins for Velero
+
+
+## Community Roles
+
+* **Users:** Members that engage with the Velero community via any medium (Slack, GitHub, mailing lists, etc.).
+* **Contributors:** Regular contributions to projects (documentation, code reviews, responding to issues, participation in proposal discussions, contributing code, etc.). 
+* **Maintainers**: The Velero project leaders. They are responsible for the overall health and direction of the project; final reviewers of PRs and responsible for releases. Some Maintainers are responsible for one or more components within a project, acting as technical leads for that component. Maintainers are expected to contribute code and documentation, review PRs including ensuring quality of code, triage issues, proactively fix bugs, and perform maintenance tasks for these components.
```

Comment:
Should technical/feature leads be another role, a step forward from Contributors towards becoming a maintainer?

Reply:
Can you be more specific as to what makes that role more distinct from a contributor or maintainer?

Reply:
It is like giving a category to the Feature leads (Stake holders) that are mentioned here: https://github.com/vmware-tanzu/velero/blob/a86079220173c4e7f5f9dead9aa5857ba77ba758/MAINTAINERS.md#velero-contributors--stakeholders

The journey could be: 
Contributors -> Leads/Stake holders -> Maintainers

As the load/backlog on the Maintainers increases, Maintainers could nominate one of the Contributors who are familiar with a certain feature/area and have significantly contributed to that area to become a Lead/Stake Holder

Reply:
@kmova that's a good suggestion. others what do you think?
cc: @vmware-tanzu/velero-maintainers

Reply:
I think this is worth talking about "face-to-face" in our community meeting. 

Initially my question is, other than on-paper recognition (which I'm all for), how would this role be different than a maintainer, in practical terms?